controls/srg_gpos/SRG-OS-000480-GPOS-00227.yml

```diff
@@ -163,11 +163,8 @@ controls:
             - xwindows_runlevel_target
 
             # dconf
-            - dconf_db_up_to_date
```

Comment:
Isn't this rule important to keep the dconf binary db up to date?

Reply:
This rule was only in CIS, not in the STIG before. https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/gnome/dconf_db_up_to_date/rule.yml